documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -18,10 +18,10 @@ image::payloads-diagram.svg[opts=interactive, width=100%]
 
 For understanding the construction rules, we define the following terms:
 
-* A Property's _leaf Characteristic_ means the Property's Characteristic if it is not a Constraint,
+* A Property's _trait Characteristic_ means the Property's Characteristic if it is not a Constraint,
```

Comment:
This sentence needs to be adjusted, because since BAMM 2.0.0 multiple Constraints are not modeled as chains any more, i.e. there is no transitively following a Constraint's `baseCharacteristic` (because Constraints don't have a baseCharacteristic any more, instead the Trait defines the baseCharacteristic).